Undefined reference to ThrowStdOutOfRange and RawLog

I am trying to build a simple example using `GNU 7.3.0`

```cpp
#include <iostream>
#include <string>
#include <vector>

#include <absl/strings/str_join.h>

int main() {
    std::vector<std::string> v = {"foo","bar","baz"};
    std::string s = absl::StrJoin(v, "-");
    absl::StrAppend(&s, 5);
    std::cout << "Joined string: " << s << "\n";
    return 0;
}
```

And get linker errors

```bash
/lib/libabsl_strings.a(numbers.cc.o): In function `absl::lts_2018_12_18::SimpleAtof(absl::lts_2018_12_18::string_view, float*)':
numbers.cc:(.text+0x518): undefined reference to `absl::lts_2018_12_18::base_internal::ThrowStdOutOfRange(char const*)'
/lib/libabsl_strings.a(numbers.cc.o): In function `absl::lts_2018_12_18::SimpleAtod(absl::lts_2018_12_18::string_view, double*)':
numbers.cc:(.text+0x8c8): undefined reference to `absl::lts_2018_12_18::base_internal::ThrowStdOutOfRange(char const*)'
/lib/libabsl_strings.a(numbers.cc.o): In function `absl::lts_2018_12_18::SimpleAtob(absl::lts_2018_12_18::string_view, bool*)':
numbers.cc:(.text+0xb29): undefined reference to `absl::lts_2018_12_18::raw_logging_internal::RawLog(absl::lts_2018_12_18::LogSeverity, char const*, int, char const*, ...)'
/lib/libabsl_strings.a(ascii.cc.o): In function `absl::lts_2018_12_18::RemoveExtraAsciiWhitespace(std::string*)':
ascii.cc:(.text+0x468): undefined reference to `absl::lts_2018_12_18::base_internal::ThrowStdOutOfRange(char const*)'
```

Whereas on `Apple LLVM version 10.0.0 (clang-1000.11.45.5)` everything is ok

Even the whole list of libraries does not help

```bash
absl_dynamic_annotations
absl_internal_spinlock_wait
absl_leak_check_disable
absl_leak_check
absl_bad_optional_access
pow10_helper
absl_internal_city
absl_container
absl_bad_any_cast
absl_int128
absl_numeric
absl_base
absl_utility
absl_meta
absl_internal_throw_delegate
absl_internal_malloc_internal
absl_variant
absl_any
absl_span
absl_demangle_internal
absl_internal_debugging_internal
absl_optional
absl_symbolize
absl_stacktrace
absl_internal_examine_stack
absl_strings
absl_failure_signal_handler
absl_hash
absl_raw_hash_set
str_format_extension_internal
absl_time
str_format_internal
absl_str_format
absl_synchronization
```

How do you think where an issue is?

--- a/conanfile.py
+++ b/conanfile.py
@@ -38,46 +38,14 @@ class AbseilConan(ConanFile):
 
     def package(self):
         self.copy("LICENSE", dst="licenses")
-        self.copy("*.h", dst="include", src="absl")
-        self.copy("*.inc", dst="include", src="absl")
+        self.copy("*.h", dst="include", src=".")
+        self.copy("*.inc", dst="include", src=".")
         self.copy("*.a", dst="lib", src=".", keep_path=False)
         self.copy("*.lib", dst="lib", src=".", keep_path=False)
 
     def package_info(self):
-        self.cpp_info.libs = ["absl_base",
-                              "absl_synchronization",
-                              "absl_strings",
-                              "absl_symbolize",
-                              "absl_malloc_internal",
-                              "absl_time",
-                              "absl_strings",
-                              "absl_base",
-                              "absl_dynamic_annotations",
-                              "absl_spinlock_wait",
-                              "absl_throw_delegate",
-                              "absl_stacktrace",
-                              "absl_int128",
-                              "absl_span",
-                              "test_instance_tracker_lib",
-                              "absl_stack_consumption",
-                              "absl_bad_any_cast",
-                              "absl_hash",
-                              "str_format_extension_internal",
-                              "absl_failure_signal_handler",
-                              "absl_str_format",
-                              "absl_numeric",
-                              "absl_any",
-                              "absl_optional",
-                              "absl_container",
-                              "absl_debugging",
-                              "absl_memory",
-                              "absl_leak_check",
-                              "absl_meta",
-                              "absl_utility",
-                              "str_format_internal",
-                              "absl_variant",
-                              "absl_examine_stack",
-                              "absl_bad_optional_access",
-                              "absl_algorithm"]
         if self.settings.os == "Linux":
-            self.cpp_info.libs.append("pthread")
+            self.cpp_info.libs = ["-Wl,--start-group"]
+        self.cpp_info.libs.extend(tools.collect_libs(self))
+        if self.settings.os == "Linux":
+            self.cpp_info.libs.extend(["-Wl,--end-group", "pthread"])

